glide.yaml

```diff
@@ -166,8 +166,8 @@ import:
     version: 4eba002a5eaea69cf8d235a388fc6b65ae68d2dd
 
   # START_PROMETHEUS_DEPS
-  - package: github.com/prometheus/prometheus
-    version: ~2.12.0
+  - package: github.com/prometheus/prometheus 
+    version: ~2.12.0 
```

Comment:
Think we can revert both `glide.yaml` and `glide.lock` changes yeah? 

Seems like nothing changed in the `glide.yaml` just formatting and I don't see new deps, etc.

Reply:
Ah yeah, think there was some weirdness involved here from updating my old branch